app/stack/route.js

```diff
@@ -14,17 +14,17 @@ export default Ember.Route.extend({
     return tokens.join(' - ');
   },
 
+  setupController: function(controller, model) {
+    controller.set('model', model);
+    controller.set('organizations', this.store.find('organization'));
```

Comment:
@bantic Thanks for the explanation.  Since they are only used for the sidebar, I think in this instance it's good to have organizations and stacks loaded async so as to not block the rendering of the primary content.  Please let me know if there is a better way to handle this.